dissociate-token-via-ethers/index.js

```diff
@@ -0,0 +1,32 @@
+
+import ethers from 'ethers';
+import { TokenId,} from '@hashgraph/sdk';
+
+/* Dissociate an account from the provided HTS token id by 
+  treating HTS tokens as if they were ERC20/ERC721 tokens. 
+  This allows us to call the dissociate functino directly on the HTS token.
+*/
+async function dissociateToken() {
+  // set up your ethers provider
+  const provider = new ethers.providers.Web3Provider(window.ethereum, "any");
```

Comment:
This seems to be set up as a NodeJs script, apart from this line, in particular `window.ethereum`, which would work in a browser. Which one is the intent?

If the intent is browser, then I suggest including an `index.html` file which loads this script.

If the intent is NodeJs, then I suggest reading an operator account (from `.env`), and initialising the provider using a JSON-RPC endpoint (e.g. localhost for RPC-relay)

Reply:
I have addressed this comment and added an HTML file